test/servertester.go

```diff
@@ -273,3 +273,9 @@ func (st *serverTester) writeRSTStream(streamID uint32, code http2.ErrCode) {
 		st.t.Fatalf("Error writing RST_STREAM: %v", err)
 	}
 }
+
+func (st *serverTester) writeGoAwayPing(ack bool, data [8]byte) {
```

Comment:
`writePing`?  There's nothing about this ping that implies any relation to goaway besides the payload which is parameterized in `data`.

Reply:
Oh lol fair point good catch. Changed.